test/verify/check-storage-mdraid

```diff
@@ -152,81 +134,77 @@ class TestStorageMdRaid(storagelib.StorageCase):
-            b.wait_in_text(info_field("State"), "Not running")
+            b.wait_text(self.card_desc("MDRAID device", "State"), "Not running")
             self.raid_default_action("Start")
-            b.wait_in_text(info_field("State"), "Running")
+            b.wait_text(self.card_desc("MDRAID device", "State"), "Running")
 
-        # Stopping and starting the array will create the expected
-        # symlink in /dev/md/.
+        # Stopping and starting the array should not change the device.
         #
-        b.wait_text(info_field("Device"), "/dev/md/raid0")
+        b.wait_text(self.card_desc("MDRAID device", "Device"), "/dev/md/raid0")
 
         # Partitions also get symlinks in /dev/md/, so they should
         # have names like "/dev/md/raid0p1".
-        if self.storaged_version >= [2, 6, 3]:
-            part_prefix = "/dev/md/raid0p"
-        else:
-            # Older storaged versions don't pick them up as the preferred
-            # device, however, so we will see "/dev/md127p1", for example.
-            # (https://github.com/storaged-project/storaged/issues/98)
-            part_prefix = dev + "p"
+        #
+        part_prefix = "md/raid0p"
 
         # Create Partition Table
-        b.click('button:contains(Create partition table)')
+        self.click_card_dropdown("MDRAID device", "Create partition table")
         self.dialog({"type": "gpt"})
-        self.content_row_wait_in_col(1, 0, "Free space")
+        b.wait_text(self.card_row_col("GPT partitions", 1, 1), "Free space")
 
         # Create first partition
-        self.content_row_action(1, "Create partition")
+        self.click_dropdown(self.card_row("GPT partitions", 1), "Create partition")
         self.dialog({"size": 20,
                      "type": "ext4",
                      "mount_point": "/foo1",
                      "name": "One"},
                     secondary=True)
-        self.content_row_wait_in_col(1, 2, "ext4 filesystem")
-        self.content_row_wait_in_col(1, 1, part_prefix + "1")
+        b.wait_text(self.card_row_col("GPT partitions", 1, 2), "ext4 filesystem")
+        b.wait_text(self.card_row_col("GPT partitions", 1, 1), part_prefix + "1")
 
         # Create second partition
-        self.content_row_action(2, "Create partition")
+        self.click_dropdown(self.card_row("GPT partitions", 2), "Create partition")
         self.dialog({"type": "ext4",
                      "mount_point": "/foo2",
                      "name": "Two"})
+        b.wait_text(self.card_row_col("GPT partitions", 2, 2), "ext4 filesystem")
+        b.wait_text(self.card_row_col("GPT partitions", 2, 1), part_prefix + "2")
+        b.wait_not_present(self.card_row("GPT partitions", name="Free space"))
 
-        self.content_row_wait_in_col(2, 2, "ext4 filesystem")
-        self.content_row_wait_in_col(2, 1, part_prefix + "2")
-        b.wait_not_in_text("#detail-content", "Free space")
+        b.assert_pixels('body', "page",
```

Comment:
I checked the pixel tests, and this one caught my eye: https://github.com/cockpit-project/pixel-test-reference/compare/14af1609e4be0b8cc39a3c1955abb103e0ab0d73..925f0d5c4f2f5d1ec693f7a1204e7f066c7bbe2e#diff-7f841b4327cc14b47fa911a423c8c1d885eec0a0879c98209ef20af80725d7d3

There is an arrow at the bottom which doesn't point to anything. That feels like a bug?

Reply:
This is outdated, there are no arrows anymore. In this pixel reference, there should be a filesystem card or partition table card below the mdraid device card, but it is scrolled out of view.